akp/data_source_akp_cluster_schema.go

```diff
@@ -201,3 +211,114 @@ func getManagedClusterConfigDataSourceAttributes() map[string]schema.Attribute {
+			Computed:    true,
+			Attributes:  getAppControllerAutoScalingConfigDataSourceAttributes(),
+		},
+		"repo_server": schema.SingleNestedAttribute{
+			Description: "Repo Server auto scaling config",
+			Computed:    true,
+			Attributes:  getRepoServerAutoScalingConfigDataSourceAttributes(),
+		},
+	}
+}
+
+func getCustomAgentSizeConfigDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"application_controller": schema.SingleNestedAttribute{
+			Description: "Application Controller custom agent size config",
+			Computed:    true,
+			Attributes:  getAppControllerCustomAgentSizeConfigDataSourceAttributes(),
+		},
+		"repo_server": schema.SingleNestedAttribute{
+			Description: "Repo Server custom agent size config",
+			Computed:    true,
+			Attributes:  getRepoServerCustomAgentSizeConfigDataSourceAttributes(),
+		},
+	}
+}
+
+func getAppControllerCustomAgentSizeConfigDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"memory": schema.StringAttribute{
+			Description: "Memory",
+			Computed:    true,
+		},
+		"cpu": schema.StringAttribute{
+			Description: "CPU",
+			Computed:    true,
+		},
+	}
+}
+
+func getRepoServerCustomAgentSizeConfigDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"memory": schema.StringAttribute{
+			Description: "Memory",
+			Computed:    true,
+		},
+		"cpu": schema.StringAttribute{
+			Description: "CPU",
+			Computed:    true,
+		},
+		"replica": schema.Int64Attribute{
+			Description: "Replica",
+			Computed:    true,
+		},
+	}
+}
+
+func getAppControllerAutoScalingConfigDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"resource_minimum": schema.SingleNestedAttribute{
+			Description: "Resource minimum",
+			Computed:    true,
+			Attributes:  getResourcesDataSourceAttributes(),
+		},
+		"resource_maximum": schema.SingleNestedAttribute{
+			Description: "Resource maximum",
+			Computed:    true,
+			Attributes:  getResourcesDataSourceAttributes(),
+		},
+	}
+}
+
+func getRepoServerAutoScalingConfigDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"resource_minimum": schema.SingleNestedAttribute{
+			Description: "Resource minimum",
+			Computed:    true,
+			Attributes:  getResourcesDataSourceAttributes(),
+		},
+		"resource_maximum": schema.SingleNestedAttribute{
+			Description: "Resource maximum",
+			Computed:    true,
+			Attributes:  getResourcesDataSourceAttributes(),
+		},
+		"replica_maximum": schema.Int64Attribute{
+			Description: "Replica maximum",
+			Computed:    true,
+		},
+		"replica_minimum": schema.Int64Attribute{
+			Description: "Replica minimum",
+			Computed:    true,
+		},
+	}
+}
+
+func getResourcesDataSourceAttributes() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"mem": schema.StringAttribute{
```

Comment:
based on @junzebao 's comment, this also need to be "memory“？

Reply:
Revised. Was only revised custom agent size config, and now I've also revised auto agent size config. Thank you!